radlab-ui/webapp/src/pages/api/deployments/index.ts

```diff
@@ -67,6 +69,11 @@ const createDeployment = async (
     console.error(error)
   }
 
+  await configureEmailAndSend(
+    "RAD Lab Module has been created for you!",
+    response,
+  )
```

Comment:
These probably should've been done as Pub/Sub topic publishes, and the email sending as a Subscriber. If we have to do another async feature like this, we will rewrite both to Pub/Sub.

Reply:
No action needed at this time.